glib/src/subclass/object.rs

```diff
@@ -806,8 +619,11 @@ mod test {
         })
         .expect("Failed to connect on 'create-string'");
 
+        let signal_id = SignalId::lookup("create-string", SimpleObject::static_type())
+            .expect("Signal crate-string not found");
```

Comment:
You can also get it directly from `SimpleObject::signals()[0]` :)

Reply:
I'm not sure about this one, ObjectImpl is implemented for the private struct, you can't call signals on the public one. see
```

error[E0599]: no function or associated item named `signals` found for struct `subclass::object::test::SimpleObject` in the current scope
   --> glib/src/subclass/object.rs:622:39
    |
622 |         let signal_id = SimpleObject::signals()[0];
    |                                       ^^^^^^^ function or associated item not found in `subclass::object::test::SimpleObject`
    | 
   ::: glib/src/object.rs:640:9
    |
640 |         pub struct $name($crate::object::ObjectRef);
    |         -------------------------------------------- function or associated item `signals` not found for this
    ```

Reply:
Oh yeah, you need to get the other struct :)

Reply:
Sorry, I feel stupid.